### PR TITLE
[WIP] demo about versioning config

### PR DESCRIPTION
**Pull request by superyyrrzz**
Related: https://github.com/OfficeDev/office-js-docs-pr/pull/675

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c1fa2b2](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/c1fa2b2ec6a7ee7053c1c1077a5ba1efef420ed7)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 2 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c1fa2b2](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/c1fa2b2ec6a7ee7053c1c1077a5ba1efef420ed7)_: 

:clock10: **Full build**: average full build time is _4 min(s) 47 sec(s)_ [calculated from last _2_ build(s)]

> Note: A full build takes more time than an incremental build.  This is the expected behavior of the build system.

File: docs/docfx.json [hunk 1]
```diff
@@ -90,13 +152,17 @@
...
         ],
         "src": "docs-ref-conceptual",
-        "version": "office-js",
         "dest": "office"
       }
     ],
-    "versions": {
-      "office-js": {
-        "dest": "office-js"
+    "groups": {
```
Comment (superyyrrzz) [inline docs/docfx.json]: `groups` names should match `moniker_ranges` in `.openpublishing.publish.config.json`

**Comment by o365devx** [comment]
OPS Build status updates of commit _[c1fa2b2](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/c1fa2b2ec6a7ee7053c1c1077a5ba1efef420ed7)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :warning:Warning | | [Details](#user-content-)
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/docfx.json](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docs-ref-autogen/outlook_1_4/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docs-ref-autogen/outlook_1_5/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docs-ref-autogen/toc.yml) | :white_check_mark:Succeeded | |

<a name=""></a>
### []()
 - **[Warning]** Multiple input files would generate to the same output path overwriting each other. Please rename at least 1 of following input files to ensure that there will be only one file to generate to the output path: &quot;docs-ref-autogen/office/office.index.yml, docs-ref-autogen/outlook_1_4/office.index.md&quot;.
 - **[Warning]** Multiple input files would generate to the same output path overwriting each other. Please rename at least 1 of following input files to ensure that there will be only one file to generate to the output path: &quot;docs-ref-autogen/office/office.index.yml, docs-ref-autogen/outlook_1_4/office.index.md&quot;.
 - **[Warning]** Multiple input files would generate to the same output path overwriting each other. Please rename at least 1 of following input files to ensure that there will be only one file to generate to the output path: &quot;docs-ref-autogen/office/office.index.yml, docs-ref-autogen/outlook_1_5/office.index.md&quot;.
 - **[Warning]** Multiple input files would generate to the same output path overwriting each other. Please rename at least 1 of following input files to ensure that there will be only one file to generate to the output path: &quot;docs-ref-autogen/office/office.index.yml, docs-ref-autogen/outlook_1_5/office.index.md&quot;.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C10%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201901100411024503-285%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kXchb8PT2pLQagYHmOXGquPaABo2wWmLARhnPahsFxI%3D&st=2019-01-10T04%3A10%3A44Z&se=2019-02-10T04%3A15%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[feacb27](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/feacb271ddbfa1bd7f36e638b79f17e64f278356)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 2 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[feacb27](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/feacb271ddbfa1bd7f36e638b79f17e64f278356)_: 

:clock10: **Full build**: average full build time is _4 min(s) 28 sec(s)_ [calculated from last _3_ build(s)]

> Note: A full build takes more time than an incremental build.  This is the expected behavior of the build system.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[544295d](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/544295dca8d6553ec999077b44a74674897af95c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[544295d](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/544295dca8d6553ec999077b44a74674897af95c)_: 

:clock10: **Full build**: average full build time is _4 min(s) 2 sec(s)_ [calculated from last _4_ build(s)]

> Note: A full build takes more time than an incremental build.  This is the expected behavior of the build system.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[544295d](https://github.com/OfficeDev/office-js-docs-reference/pull/285/commits/544295dca8d6553ec999077b44a74674897af95c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[docs/docfx.json](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_4/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docs-ref-autogen/outlook_1_4/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/outlook_1_5/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docs-ref-autogen/outlook_1_5/toc.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/toc.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/renzeyu/docs/docs-ref-autogen/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201901180651080232-285%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kpoDInILafL%2BY8wlode1PugvqUvfochdC0r1mi3opnc%3D&st=2019-01-18T06%3A51%3A29Z&se=2019-02-18T06%3A56%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by lindalu-MSFT** [comment]
Hi @jmazzotta What's the status with this topic? Is it ready to merge and publish? LMK if there's anything I can help with! I noticed it's been hanging out for a few months and didn't want it overlooked. I think there's a few merge conflicts that you'll need to fix, or I can if you want me to look into it. Thanks, lindalu

**Comment by superyyrrzz** [comment]
@lindalu-MSFT @jmazzotta This is only for demo purpose. I will close it. Thank you for reminder!

**Pull request closed without merging**